Python 3.10.0 (tags/v3.10.0:b494f59, Oct  4 2021, 19:00:18) [MSC v.1929 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license()" for more information.

======================== RESTART: C:/Python310/sets1.py ========================
<class 'dict'>
<class 'set'>
<class 'set'>
False
{8.9, 'a', 4, None}
{(2, 3), 'rosjini', 'a'}
2048911167264
2048911167264
True
False
{(2, 3), 'a', 4, 8.9, 'rosjini', 30, None}
{'a'}
{8.9, 4, 30, None}
{(2, 3), 4, 8.9, 'rosjini', 30, None}
False
False
Traceback (most recent call last):
  File "C:/Python310/sets1.py", line 41, in <module>
    print(a.remove(None))
KeyError: None
